test/system/010-images.bats

```diff
@@ -309,4 +309,15 @@ Deleted: $pauseID"
     is "$output" "" "Should print no output"
 }
 
+@test "podman images - commit docker with comment" {
+    run_podman run --name my-container -itd $IMAGE sleep 1d
+    run_podman 125 commit -m comment my-container my-test-image
+    assert "$output" == "Error: messages are only compatible with the docker image format (-f docker)" "podman should fail unless docker format"
+    run_podman commit my-container --format docker -m comment my-test-image
+    assert "$output" !~ "COMMENT is not supported for OCI image format" "No warning message when using --format docker"
```

Comment:
In the interests of futureproofing, I have a moderate preference for not hardcoding error messages. Instead, add `-q` to the commit and confirm that output is a 64-character hex string:
```diff
diff --git a/test/system/010-images.bats b/test/system/010-images.bats
index 8a620770b..2a2296db4 100644
--- a/test/system/010-images.bats
+++ b/test/system/010-images.bats
@@ -313,8 +313,9 @@ Deleted: $pauseID"
     run_podman run --name my-container -itd $IMAGE sleep 1d
     run_podman 125 commit -m comment my-container my-test-image
     assert "$output" == "Error: messages are only compatible with the docker image format (-f docker)" "podman should fail unless docker format"
-    run_podman commit my-container --format docker -m comment my-test-image
-    assert "$output" !~ "COMMENT is not supported for OCI image format" "No warning message when using --format docker"
+    run_podman commit -q my-container --format docker -m comment my-test-image
+    assert "$output" =~ "^[0-9a-f]{64}\$" \
+           "Output is a commit ID, no warnings or other output"
 
     run_podman rmi my-test-image
     run_podman rm my-container --force -t 0
```
Not a blocker though.

Reply:
SGTM
@rhatdan PTAL